.cirrus.yml

```diff
@@ -116,9 +116,9 @@ mac_task:
         PLUGINS: ALL;-curlget # The curlget tests fail: https://github.com/ElektraInitiative/libelektra/issues/3382
 
   install_script:
-    - > # Install Homebrew casks
-      brew cask install java oclint
     - | # Install Homebrew formulas
+      brew install openjdk
```

Comment:
This will most probably not work. You need to use `brew cask install openjdk` here, and `brew cask install oclint` below.

Reply:
Yes, it did not work. I thought cask is for GUI? I now tried with a tap.

Reply:
>  I thought cask is for GUI?

That is the case a lot of the time. More generally though Homebrew Cask is a place for (non-open) software that already provides  an installer (`.pkg`) or contains a normal (GUI) application (`.app`). Seems like the Homebrew people are actively working on their infrastructure. Now

```sh
brew install openjdk
```

works and

```sh
brew cask install openjdk
```

(which – as far as I remember – worked a few hours ago)  does not any more.